Jenkinsfile

```diff
@@ -492,3 +462,36 @@ def generateStepForWindows(Map v = [:]){
     }
   }
 }
+
+def convergeCoverage() {
+  if (env.ONLY_DOCS == "false") {
+    deleteDir()
+    unstash('source')
+    unstash('packages')
```

Comment:
This is required, the `opbeans` stage deletes the workspace:
- https://github.com/elastic/apm-agent-python/blob/a577638c6ec4520560e61fc01ad5f1eb594f97e4/Jenkinsfile#L290